model/Core/Datatypes/SpdxId.md

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SpdxId
```

Comment:
No need for this type, which is the same as `xsd:anyURI`.

Reply:
There is a need because SPDX has anyURI datatypes that are not SpdxIds (such as content identifier, package URL, home page, etc) that are not SpdxIds.  A homepage cannot be used as an Element id.

Reply:
As in #407, the reason for defining an SpdxId datatype for xsd:anyURL is to distinguish IRIs that are SpdxIds (e.g., declaredLicense, concludedLicense, originatedBy, suppliedBy) from IRIs that are not SpdxIds (e.g., downloadLocation, packageUrl, homepage). These are all of type xsd:anyURL but they are serialized differently  - namespaceMap is not used to compress IRIs that are not SpdxIds.

Look at [Package](https://github.com/spdx/spdx-3-serialization-prototype-playground/blob/main/model/generated_20230719T155944Z/Software/Package.md) for a concrete example, and compare downloadLocation with concludedLicense. 

Naming the SpdxId datatype, just like naming the DateTime datatype, obviously adds to what would be in the model without them.  No rationale has been presented for how naming simple Datatypes is harmful, and the names are helpful.  

Just do it.

Reply:
@davaya - Can we remove the SPDX ID from this PR?  You already have this documented in a separate issue or pull request and there doesn't seem to be any question on the MediaType.  I'd like to get the MediaType merged in.  We can continue the debate on SPDX ID in a separate thread.

Reply:
Will do. Do you prefer just the 10 registries for the first part, or the ABNF string?

Reply:
Include "registries vs. type names" decision on Aug 1 tech team meeting.